### ISSUE TITLE
\ being added to built in functions in 0.7.54, works correctly in 0.7.53

### ISSUE THREAD

**Issue by phpfui** [rectorphp/rector#3814]
# Bug Report

| Subject           | Details           |
| -------------- | --------------|
| Rector version | 0.7.54           |

Standard built in functions in PHP are being prepended with the root namespace causing PHPStan to barf.  For example, count becomes \count.  While I don't think that is technically illegal PHP, it is not needed and causes unexpected T_NS_SEPARATOR errors in PHPStan.  Works in 0.7.53.

## Minimal PHP Code Causing Issue

```php
class Comet
{
    function name()
    {
       ob_start();
    }
}
```

## Rector.yaml (can't seem to convert it to PHP yet)

```YAML
parameters:
    php_version_features: '5.6'
services:
    Rector\Legacy\Rector\Include_\RemoveIncludeRector: null
```

## Expected Behaviour

Nothing should change in file.  Rector returns \ob_start();  Code needs to be in method in class.  Stand alone function call, or function call in function work correctly.

**Comment by TomasVotruba** [comment]
Interesting bug, wow.

Could you narrow it to specific commit?

**Comment by phpfui** [comment]
I'll try. Battling mismatched php tags in views causing syntax errors now,
but a break might be good.

**Comment by phpfui** [comment]
Looks like the first commit after 0.7.53 is bad, but not sure why.  It does update other libraries.

  - Updating doctrine/inflector (1.4.x-dev 4650c8b => dev-master e579cfd):  Checking out e579cfde91
  - Updating phpstan/phpstan (0.12.32 => dev-master 807fad8):  Checking out 807fad8274
  - Updating nikic/php-parser (v4.5.0 => v4.6.0): Loading from cache
  - Updating phpstan/phpstan-phpunit (0.12.11 => dev-master 20f23c9):  Checking out 20f23c988c
  - Updating rector/rector (v0.7.53 => dev-master 407e0c4):  Checking out 407e0c4d33

From composer.json:
"rector/rector": "dev-master#407e0c4d3314d0291c4073010abf28e608d18047"

**Comment by TomasVotruba** [comment]
That commit is realted to this PR: https://github.com/rectorphp/rector/pull/3773/files

**Comment by phpfui** [comment]
I did notice that 0.7.53 is doing the same thing in some of my source files, but not all.  I'll keep investigating and post an update when I know more.  But for sure 0.7.53 is OK with the example code and 0.7.54 and this commit both do the wrong thing.  The commit seems to barf and not change the file, where as 0.7.54 modifies the file and barfs.

I also noticed a problem will a deleted <?php tag using the RemoveIncludeRector, but will open a separate issue when I figure that out.  I don't believe they are related.
